packages/foreman/foreman/foreman.spec

```diff
@@ -705,6 +707,8 @@ plugins required for Foreman to work.
   done
   # script content
   sed -ri 'sX/usr/bin/rakeX%{scl_rake}X' extras/dbmigrate script/foreman-rake
+  sed -i '2isource scl_source enable tfm' script/foreman-puma-status
+  when: foreman_installer_upgrade == True
```

Comment:
Copy paste error?
```suggestion
```

Reply:
Huh. I don't know even know how that little bit of magic made it in there.